notesapi/v1/tests/test_views.py

```diff
@@ -65,6 +66,16 @@ def _create_annotation(self, **kwargs):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         return response.data.copy()
 
+    def _create_annotation_without_assert(self, **kwargs):
+        """
+        Create annotation but do not check for 201
+        """
+        opts = self.payload.copy()
+        opts.update(kwargs)
+        url = reverse('api:v1:annotations')
+        response = self.client.post(url, opts, format='json')
+        return response
```

Comment:
@ehteshamkafeel Instead of creating a new method we can use the existing `_create_annotation` and use a kwarg to decide whether we need to check the http status_code.